app/controllers/appointment_summaries_controller.rb

```diff
@@ -15,11 +15,13 @@ def create
   def show
     appointment_summary = AppointmentSummary.find(params[:id])
     output_document = OutputDocument.new(appointment_summary)
+    output_document_pdf = output_document.pdf
+    OutputDocumentMailer.issue(appointment_summary, output_document_pdf).deliver_later
```

Comment:
Good point about the action @gjvis , this is redirected to after the `AppointmentSummary` is created so perhaps it's better in the create action or elsewhere